table.go

```diff
@@ -125,6 +125,25 @@ func (f *File) AddTable(sheet string, table *Table) error {
 	return f.addContentTypePart(tableID, "table")
 }
 
+func (f *File) GetTables(sheet string) ([]Table, error) {
+	var tables []Table
+	ws, err := f.workSheetReader(sheet)
+	if err != nil {
+		return tables, err
+	}
+	for _, tbl := range ws.TableParts.TableParts {
```

Comment:
We need to checking table parts if nil before access and range it.

Reply:
Thanks for the review. I've added check statements for it.